sass/atoms/_base.scss

```
@@ -5,6 +5,7 @@ html {
   background-color: $white;
   box-sizing: border-box;
   color: $text-color;
+  scroll-behavior: smooth;
```

There is an issue in some browsers where setting smooth scrolling affects user experience when you do `Ctrl + F` or tab navigation.

The solution is to use the smooth rule only when the focus is within the document.

Reference: https://css-tricks.com/fixing-smooth-scrolling-with-find-on-page/

```suggestion
}

/* https://css-tricks.com/fixing-smooth-scrolling-with-find-on-page/ */
html:focus-within {
  scroll-behavior: smooth;
}
```

---

Yep, that will work

---

Thanks for the noticing and providing the solution. This will improve the flow to be much more faster for readers.